src/Widget/Controller/WidgetController.php

```diff
@@ -332,8 +332,8 @@ private function getWidget(WidgetPageInterface $widgetPage, $widgetId)
     public function getRegionAutocompleteResult(Request $request, $searchString, $language = 'nl')
     {
         $matches = $this->regionService->getAutocompletResults($searchString, $language);
-        // Sort $matches according levenshtein distance
-        $matches = $this->regionService->sortByLevenshtein($matches, $searchString);
+        // Sort the matches on nis which was added as key of the matches array.
+        \ksort($matches);
```

Comment:
is this backslash necessary?

```suggestion
        ksort($matches);
```

Reply:
It is a PHPStorm thing, it automatically adds a slash when a function from the global namespace is used. I removed the slash.